scopus-import/src/main/java/no/sikt/nva/scopus/conversion/AffiliationGenerator.java

```diff
@@ -82,39 +75,49 @@ private List<CorporationWithContributors> fillCristinOrganizationData(
     }
 
     private CorporationWithContributors addCristinOrganisationData(CorporationWithContributors corporation) {
-        return nonNull(corporation.getCristinOrganizationId())
-                   ? fetCristinOrganizationsById(corporation)
-                   : searchForCristinOrganizationByName(corporation);
+        var cristinOrganization = fetchCristinOrganizationsById(corporation)
+            .or(() -> searchForCristinOrganizationByName(corporation))
+            .or(() -> searchForCristinOrganizationByCountry(corporation));
+
+        return cristinOrganization.isPresent()
+                   ? corporation.copy().withCristinCorporations(List.of(cristinOrganization.get())).build()
+                   : corporation;
+    }
+
+    //Countries are stored as organizations in Cristin, we use the same endpoint
+    //for fetching countries as we do with organizations
+    private Optional<CristinOrganization> searchForCristinOrganizationByCountry(
```

Comment:
Maybe call this method `searchForCountry` since the organisation in question is a country…I'd also take a moment to consider what an Organization (big-O) is in the ontology "Administrative grouping of people working towards a shared goal", which seems at odds with the current use as I don't think any of the current organizations can be said to be "working towards a shared goal", but "Administrative grouping of people" covers all current cases. Update the ontology accordingly pls.